packages/components/src/internal/popup-utils.ts

```diff
@@ -104,14 +107,27 @@ export const computePopupPosition = (args: ComputePopupPositionArgs): ComputePop
 
 type BoundingRect = Size & Position;
 
-const toBoundingRect: (domRect: DOMRect) => BoundingRect = _.pick([
-  'width',
-  'height',
-  'top',
-  'right',
-  'bottom',
-  'left',
-]);
+const toBoundingRect = (domRect: DOMRect): BoundingRect => {
```

Comment:
These two new functions are both special cases of generic Lodash functions, tailored to the use cases in this file.